### PR TITLE
Migrate sort button

### PR DESCRIPTION
**Pull request by mdelez**
closes https://dasch.myjetbrains.com/youtrack/issue/DSP-320

### CODE REVIEW

**Comment by mdelez** [comment]
Screen Shot 2020-06-03 at 11 00 25

In addition to moving the sort button, I also decided to make a playground for these action components :)

File: projects/dsp-ui/src/lib/action/components/sort-button/sort-button.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        TestBed.configureTestingModule({
+            imports: [
+                MatIconModule,
+                MatMenuModule,
+                BrowserAnimationsModule
+            ],
+            declarations: [
+                SortButtonComponent,
+                TestHostComponent,
+                SortByPipe]
```
Comment (flavens) [inline projects/dsp-ui/src/lib/action/components/sort-button/sort-button.component.spec.ts:101]: @mdelez why do you still use the sort-by pipe marked as deprecated? would it make sense to use the new sorting service instead (as explained in the sort-by pipe annotation)?

Comment (kilchenmann) [reply projects/dsp-ui/src/lib/action/components/sort-button/sort-button.component.spec.ts:101]: same question here 😉

**Comment by mdelez** [comment]
> LGTM

You guys are fast! Thanks!! @flavens @kilchenmann

**Pull request merged**